Articles/Dashboards/extending.md

```diff
@@ -0,0 +1,319 @@
+# Dashboards
+Each section of the Umbraco backoffice has its own set of default dashboards.
+
+The dashboard area of Umbraco is used to display an 'editor' for the selected item in the tree. If no item is selected, for example when the section 'first loads', then the default set of section dashboards are displayed in the dashboard area, arranged over multiple tabs.
```

Comment:
Not sure we need the quotes around "first loads", maybe to be clear "is first loaded in the browser"?